dm/ctl/dmctl.toml

```diff
@@ -1,3 +1,6 @@
 # dmctl Configuration.
 
+# rpc configuration
```

Comment:
more examples about how to write this item with different units?

Reply:
Currently we only have timeout in dmctl client, which means dmctl rpc timeout waiting response from dm-master, and timeout in dm-master, which means dm-master rpc timeout waiting response from dm-worker

Reply:
I mean, add some examples about how to write this config item's unit, like `10m`, `3h`, etc.

Reply:
added in https://github.com/pingcap/dm/pull/157/commits/c63653f3578b36be4e17314db9ae43e7d2177a3d